statistics/handle/handle.go

```diff
@@ -1944,3 +1944,39 @@ func (h *Handle) GetPredicateColumns(tableID int64) ([]int64, error) {
 	}
 	return columnIDs, nil
 }
+
+// SaveHistoryStatsToStorage saves the given table's stats data to mysql.stats_history
+func (h *Handle) SaveHistoryStatsToStorage(dbName string, tableInfo *model.TableInfo) (uint64, error) {
```

Comment:
How about renaming it to `RecordHistoricalStatsToStorage`?

Reply:
Done